### DISCUSSION TITLE
Zigbee device's battery discharge

### DISCUSSION THREAD

**Discussion by r-vit** [arendst/Tasmota#13865] [Q&A] [answered]
I've got a couple of Moes / TuYa TS0041 (https://www.zigbee2mqtt.io/devices/TS0041.html) wireless buttons, and both devices got their batteries depleted in 1 month. I used new IKEA CR2032 batteries which work quite well in other my devices.

When I checked the ZBBridge console I saw that there's a lot of messages coming from the buttons every few minutes, something like that:
```
00:16:00.830 ZIG: {"ZbEZSPReceived":"230002013419229FE2FEFF23A46004"}
00:16:00.832 ZIG: {"ZbEZSPReceived":"24003419229FE2FEFF23A46000030000"}
00:16:00.835 MQT: tele/ZBBridge/RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0x60A423FFFEE29F22","ShortAddr":"0x1934","ParentNetwork":"0x0000","JoinStatus":0,"Decision":3}}
00:16:01.046 ZIG: {"ZbEZSPReceived":"6200229FE2FEFF23A460"}
00:16:01.054 MQT: tele/ZBBridge/RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0x60A423FFFEE29F22","ShortAddr":"0x1934","PowerSource":false,"ReceiveWhenIdle":false,"Security":false}}
00:16:03.857 MQT: tele/ZBBridge/LightButtonBedroom/SENSOR = {"ZbReceived":{"LightButtonBedroom":{"Device":"0x1934","Name":"LightButtonBedroom","Power":0,"Endpoint":1,"LinkQuality":60}}}
00:16:04.676 ZIG: Auto-responder: ZbSend {"Device":"0x1934","Cluster":"0x000A","Endpoint":1,"Response":"000A/0007":691539364}
00:16:04.681 MQT: tele/ZBBridge/LightButtonBedroom/SENSOR = {"ZbReceived":{"LightButtonBedroom":{"Device":"0x1934","Name":"LightButtonBedroom","Cluster":10,"Read":[7],"ReadNames":{"LocalTime":true},"Endpoint":1,"LinkQuality":58}}}
00:16:12.936 ZIG: {"ZbEZSPReceived":"9B00229FE2FEFF23A46006"}
```

Is the ZBBridge supposed to behave like that or is it just a bad design of those buttons? Is it possible to make those buttons to report their state less often?

**Comment by ascillato2** [comment, marked as answer]
Yes, you can

You have Binded values that are being polled. You can UnBind them or change the frequency IF your device supports that command. More information about binding at https://tasmota.github.io/docs/Zigbee/#zigbee-binding

Please, also check the commands:  ( https://tasmota.github.io/docs/Commands/#zigbee )

```
ZbBindState
ZbUnbind
ZbNoAutoBind
```

Thanks.

**Comment by r-vit** [comment]
I'm not sure if the devices support that command:

```
09:14:56.388 CMD: ZbBindState LightButtonBedroom
09:14:56.394 MQT: stat/ZBBridge/RESULT = {"ZbBindState":"Done"}
09:15:04.073 ZIG: {"ZbEZSPReceived":"8000423419"}
09:15:04.080 MQT: tele/ZBBridge/RESULT = {"ZbRouteError":{"ShortAddr":"0x1934","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
```

**Reply by s-hadinger** [reply]
This message indicates that the device is offline (sleeping). Wake it up before sending the command.

**Comment by r-vit** [comment]
Got it. In my defense, I did try to wake it up but apparently by the time I was getting back from the button to the PC the device was already asleep again :). Anyway, here's the result:

```
11:22:43.568 CMD: ZBBindState LightButtonBedroom
11:22:43.574 MQT: stat/ZBBridge/RESULT = {"ZbBindState":"Done"}
11:22:43.653 MQT: tele/ZBBridge/RESULT = {"ZbBindState":{"Device":"0x1934","Name":"LightButtonBedroom","Status":0,"StatusMessage":"SUCCESS","Total":2,"Start":1,"Bindings":[{"Cluster":"0x0001","Endpoint":1,"ToDevice":"0x60A423FFFE091281","ToEndpoint":1},{"Cluster":"0x0006","Endpoint":1,"ToDevice":"0x60A423FFFE091281","ToEndpoint":1}]}}
11:22:44.625 MQT: tele/ZBBridge/LightButtonBedroom/SENSOR = {"ZbReceived":{"LightButtonBedroom":{"Device":"0x1934","Name":"LightButtonBedroom","0006!FD":"00","LidlPower":0,"Endpoint":1,"LinkQuality":31}}}
```

Should I try to unbind it now to save the battery?

**Reply by ascillato** [reply]
> Should I try to unbind it now to save the battery?

Yes, you can bind it again at any time if you want.
Also, you can change the frequency report if the device supports that.

**Comment by dbertie1** [comment]
I have had similar issues - and am trying simply increasing the reporting interval using (obviously the device needs to be woken up to receive):

ZbSend   {"Device":"Kitchen_wall_switch","Config":{"Power":{"MinInterval":3600,"MaxInterval":14400}}}

Interestingly whilst earlier switches did not report battery, the latest I have received does.  I have done the same to all and so will see if this makes a difference.
